paparazzi/paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -915,10 +913,14 @@ class PaparazziPluginTest {
 
     val snapshotsDir = File(fixtureRoot, "build/reports/paparazzi/images")
     val snapshots = snapshotsDir.listFiles()
-    assertThat(snapshots!!).hasLength(1)
+    assertThat(snapshots!!).hasLength(2)
```

Comment:
Careful here, [`.listFiles` JavaDoc](https://docs.oracle.com/javase/8/docs/api/java/io/File.html#listFiles--):

> There is no guarantee that the name strings in the resulting array will appear in any specific order; they are not, in particular, guaranteed to appear in alphabetical order.

Based on the above, think this could result in a flaky test.

Other places in this file apply `?.sortedBy { it.lastModified() }` when there's more than one file.